Sources/TableViewDriver.swift

```diff
@@ -163,8 +163,8 @@ open class TableViewDriver: NSObject {
         }
 
         self.tableView.registerViews(for: newModel)
-
-        if self.automaticDiffingEnabled {
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/plangrid/ReactiveLists/blob/95464a87927f57b19b9e9873455f3ddac4e5687c/Sources/TableViewDriver.swift#L166"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>

Reply:
cc @anayini danger + swiftlint works 😄